assets/template.svg

```diff
@@ -8,9 +8,9 @@
 <rect id="image" x="25" y="12" width="100" height="100" fill="url(#pattern0)"/>
 </g>
 </g>
-<g xmlns="http://www.w3.org/2000/svg" id="info">
-<text x="50%" font-family="Verdana" font-size="19" fill="white" y="148" text-anchor="middle">{{username}}</text>
-<text xmlns="http://www.w3.org/2000/svg" x="50%" font-family="sans-serif" font-size="12" fill="white" y="165" text-anchor="middle">{{name}}</text>
+<g id="info">
+<text x="50%" font-family="Verdana,DejaVu Sans,sans-serif" font-size="19" fill="white" y="148" text-anchor="middle">{{username}}</text>
+<text x="50%" font-family="Verdana,DejaVu Sans,sans-serif" font-size="12" fill="white" y="165" text-anchor="middle">{{name}}</text>
```

Comment:
Very good!
Just a little thing... I think would be better to not use Verdana as the first font, because it's "wider" than sans-serif and since here is the user name and it could be very large, it may show overflow...

> This is very "minimal", but can allow (theoretically) to show other 3-4 chars, I ipotizzate.

404 page before and after:

![image](https://user-images.githubusercontent.com/72039923/186250171-55fdd8cf-1469-4094-b853-21f008900b20.png)

![image](https://user-images.githubusercontent.com/72039923/186249939-8b6f2dd2-7bc3-4e45-abd3-34de5077102c.png)

Reply:
@Bellisario I see. I'm assuming you mean for the name, but not the username, since the name can potentially be rather long. In that case, for the name, I'll put it back to just sans-serif as you had it, and just leave the fall-backs for the username. DejaVu Sans is the same width as Verdana, so putting that first won't change anything. And anything you put after sans-serif will be ignored on all systems because sans-serif always has something mapped to it (just might vary by OS).